larpandoracontent/LArObjects/LArDiscreteProbabilityVector.cc

```diff
@@ -0,0 +1,237 @@
+/**
+ *  @file   larpandoracontent/LArObjects/LArDiscreteProbabilityVector.cc
+ *
+ *  @brief  Implementation of the lar discrete probability vector class
+ *
+ *  $Log: $
+ */
+
+#include "larpandoracontent/LArObjects/LArDiscreteProbabilityVector.h"
+
+#include <algorithm>
+#include <cmath>
+#include <numeric>
+
+namespace lar_content
+{
+
+template <typename TX, typename TY>
+DiscreteProbabilityVector::DiscreteProbabilityVector(const InputData<TX, TY> &inputData, const TX xUpperBound, 
+    const bool useWidths) :
```

Comment:
Extra 4 spaces indent for constructor arguments.

Reply:
Got it with a previous comment